plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java

```diff
@@ -62,14 +68,8 @@ public TestingOracleServer()
             throw new RuntimeException(e);
         }
 
-        try {
-            execInContainer("/bin/bash", "/etc/init.d/oracle-xe", "restart");
-        }
-        catch (InterruptedException | IOException e) {
-            throw new RuntimeException(e);
-        }
+        Failsafe.with(RETRY_POLICY).run(this::restart);
```

Comment:
I would rather tear the container down and create a container afresh.
https://github.com/trinodb/trino/blob/9b1b8296055d9bdc59dd05cc294e278c707932e9/plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java#L76-L77